cypress/integration/Packages/packages.ts

```diff
@@ -0,0 +1,181 @@
+/// <reference types="Cypress" />
+import {
+    ContentBuilder,
+    DocumentTypeBuilder
+  } from '../../../src';
+
+context('Packages', () => {
+    beforeEach(() => {
+        cy.umbracoLogin(Cypress.env('username'), Cypress.env('password'), false);
+      });
+
+      
+    function DeleteAllPackages(){
+        const url = 'https://localhost:44331/umbraco/backoffice/umbracoapi/package/GetCreatedPackages'
+        cy.umbracoApiRequest(url, 'GET', null).then((response) => {
+            response.forEach(function (value){
+                const requestUrl = 'https://localhost:44331/umbraco/backoffice/umbracoapi/package/DeleteCreatedPackage?packageId=' + value.id 
+                cy.umbracoApiRequest(requestUrl, 'POST', null)
+            });
+        });
+    }
```

Comment:
I'm wondering if we should make this into an actual helper method? Maybe something like `cy.umbracoEnsurePackageNameNotExists`? 

Although granted I'm not sure where it's relevant other than these specific tests? Maybe if someone wants to use this library for their own tests is might be something they want to have?